crates/wallet/src/wallet/mod.rs

```diff
@@ -19,6 +19,7 @@ use alloc::{
     sync::Arc,
     vec::Vec,
 };
+use chain::Indexer;
```

Comment:
⛏️ This should go under `use bdk_chain::`

Reply:
Good suggestion, fixed in #1785.